js/README.md

```diff
@@ -0,0 +1,83 @@
+# Fibre-JS
+
+This directory provides JavaScript bindings for [Fibre](https://github.com/samuelsadok/fibre). Its home is located [here](https://github.com/samuelsadok/fibre/tree/master/js). There's also a standalone repository for this directory [here](https://github.com/samuelsadok/fibre-js).
+
+## Current Status
+
+So far Fibre-JS was only tested to run in Chrome but it should work in any browser that [supports WebAssembly](https://caniuse.com/wasm).
+
+Fibre-JS currently supports the following transport providers:
+
+ - WebUSB ([Chrome and Edge only](https://caniuse.com/webusb))
+
+## How to use
+
+So you want your website or Electron App to connect to some resource(s) using Fibre. Let's start with a simple HTML scaffolding.
+
+```HTML
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Fibre-JS Test</title>
+    </head>
+    <body>
+        <h1 id="statusText">not connected</h1>
+        <button id="connectBtn" disabled>Connect...</button>
+    </body>
+</html>
+```
+
+Now, right before the closing `</body>`, insert a two script tags to load and open Fibre-JS.
+
+```HTML
+        <script src="fibre.js"></script>
+        <script>
+            fibreOpen().then((libfibre) => {
+                // Fibre-JS is ready to use!
+            });
+        </script>
+```
+
+This will load the files `fibre.js`, `libfibre-wasm.js` and `libfibre-wasm.wasm` from the same directory the HTML file comes from so make sure those files exist.
+
+Now that Fibre-JS is ready to use we can start discovering remote objects. For this we need to specify a filter string which tells Fibre where to look for objects. In this example we want to talk to a USB device that runs Fibre (more specfically an [ODrive](https://odriverobotics.com/)). The parameters we use here are something you can find out from reading the docs of your USB device or inspecting the device by using `lsusb` on a Linux or macOS computer.
+
+```JS
+const filter = 'usb:idVendor=0x1209,idProduct=0x0D32,bInterfaceClass=0,bInterfaceSubClass=1,bInterfaceProtocol=0';
+let onFoundObject = async (obj) => {
```

Comment:
const?

Reply:
why async?

Reply:
Yes right this can be const.
Regarding "async": there's an `await obj.vbus_voltage.read()` somewhat hidden in the status output expression.